tests/test_evaluation.py

```diff
@@ -49,78 +53,57 @@ def test_stats_for_series():
     }
 
 
-def test_run_evaluation_and_compute_aggregates():
-    def get_chat_responses(path: Path) -> dict:
-        responses = dict()
-        with jsonlines.open(path, "r") as reader:
-            for obj in reader:
-                responses[obj["question_id"]] = obj
-        return responses
+def read_responses(path: Path) -> dict:
+    with jsonlines.open(path) as reader:
+        return {obj["question_id"]: obj for obj in reader}
 
-    sample_reference_standard = yaml.safe_load(
-        (
-            Path(__file__).parent / "test_data" / "reference_standard_corpus_1.yaml"
-        ).read_text(encoding="utf-8")
-    )
-    sample_chat_responses_path = (
-        Path(__file__).parent / "test_data" / "chat_responses_1.jsonl"
-    )
 
-    evaluation_results = run_evaluation(
-        sample_reference_standard, get_chat_responses(sample_chat_responses_path)
+def test_run_evaluation_and_compute_aggregates():
+    reference_corpus = yaml.safe_load(
```

Comment:
I was under the impression that we don't like the term `corpus` and instead prefer to use the term `dataset` or `qa_dataset`.

Reply:
Replaced all occurrences by "data" or "dataset". I'm not sure anymore if "dataset" is better than "corpus" but at least it is used consistently. We can replace later.